config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
- root 'homepage#index'
+  resources :clients
+  root 'homepage#index'
+  namespace :admin do
+    root to:'admin#index'
+    resources :admin do
```

Comment:
Use :users here, change relevant controller name also.

Name which routes you need from resources helper, exclude the ones you don't want to use.

Reply:
Done.